examples/cfd/08_shallow_water_equation.ipynb

```diff
@@ -0,0 +1,8079 @@
+{
```

Comment:
<p>Maybe remove "initial" from M/N since they are time varying</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Thanks for noticing, fixed!</p>